runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -620,3 +620,10 @@ J9::ClassEnv::getROMConstantPool(TR::Compilation *comp, TR_OpaqueClassBlock *cla
    J9ConstantPool *ramCP = reinterpret_cast<J9ConstantPool *>(comp->fej9()->getConstantPoolFromClass(clazz));
    return ramCP->romConstantPool;
    }
+
+bool
+J9::ClassEnv::isValueTypeClass(TR_OpaqueClassBlock *clazz)
+   {
+   J9Class *j9class = reinterpret_cast<J9Class*>(clazz);
+   return J9_IS_J9CLASS_VALUETYPE(j9class);
+   }
```

Comment:
Sorry for the late review!  Looking at the implementations of things like `isAbstractClass`, should the signature of this method be changed to take a `TR::Compilation *` argument, and be implemented using

```
return comp->fej9()->isValueTypeClass(clazz);
```

Reply:
hmm, I thought we wanted to avoid adding more stuff into fej9 and those methods are implemented this way because of the legacy code is in fej9 already.

Reply:
let me try to PR this one in the open so that we can discuss this in the open.

Reply:
> I thought we wanted to avoid adding more stuff into fej9

Oh, I didn't realize that!  I'm happy to get rid of the method in `fej9` then.